llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1875,9 +1888,8 @@ static void removeBranchOnCondTrue(VPlan &Plan) {
           continue;
         NewOperands.push_back(Op);
       }
-      VPI->replaceAllUsesWith(B.createNaryOp(VPInstruction::ResumePhi,
-                                             NewOperands, VPI->getDebugLoc(),
-                                             VPI->getName()));
+      VPI->replaceAllUsesWith(
+          B.createScalarPhi(NewOperands, VPI->getDebugLoc(), VPI->getName()));
```

Comment:
This is the alternative to `removeIncomingValueFor(VPBB)`, on a VPPhi?

Reply:
Updated to use removeIncomingValueFor, thanks